toolchain/check/import_cpp.cpp

```diff
@@ -500,14 +502,9 @@ static auto GetDeclContext(Context& context, SemIR::NameScopeId scope_id)
       context.sem_ir().clang_decls().Get(scope_clang_decl_context_id).decl);
 }
 
-// Looks up the given declaration name in the Clang AST in a specific scope.
-// Returns the found declaration and its access. If not found, returns
-// `nullopt`. If there's not a single result, returns `nullptr` and default
-// access.
-static auto ClangLookupDeclarationName(Context& context, SemIR::LocId loc_id,
-                                       SemIR::NameScopeId scope_id,
-                                       clang::DeclarationName name)
-    -> std::optional<std::tuple<clang::NamedDecl*, clang::AccessSpecifier>> {
+static auto ClangLookup(Context& context, SemIR::NameScopeId scope_id,
+                        clang::DeclarationName name)
```

Comment:
Add a function comment.

FWIW: I find it a bit confusing that now we have both `ClangLookup()` and `ClangLookupDeclarationName()` getting `clang::DeclarationName()` as a parameter, and `ClangLookupDeclarationName()` only used when doing operator lookup.
I see there's a TODO though.